src/rusoto_dynamodbstore.rs

```diff
@@ -1259,53 +1278,15 @@ fn record_rcu(capacity: &Option<ConsumedCapacity>, span: &Span) {
     }
 }
 
-pub async fn create_default_table(client: &DynamoDbClient, table_name: &str) -> Result<()> {
-    let mut create = rusoto_dynamodb::CreateTableInput::default();
-    create.attribute_definitions = vec![
-        AttributeDefinition {
-            attribute_name: "hk".to_string(),
-            attribute_type: "B".to_string(),
-        },
-        AttributeDefinition {
-            attribute_name: "rk".to_string(),
-            attribute_type: "B".to_string(),
-        },
-        AttributeDefinition {
-            attribute_name: "rk2".to_string(),
-            attribute_type: "B".to_string(),
-        },
-    ];
-    create.key_schema = vec![
-        KeySchemaElement {
-            attribute_name: "hk".to_string(),
-            key_type: "HASH".to_string(),
-        },
-        KeySchemaElement {
-            attribute_name: "rk".to_string(),
-            key_type: "RANGE".to_string(),
-        },
-    ];
-    create.local_secondary_indexes = Some(vec![LocalSecondaryIndex {
-        index_name: "rk2".to_string(),
-        key_schema: vec![
-            KeySchemaElement {
-                attribute_name: "hk".to_string(),
-                key_type: "HASH".to_string(),
-            },
-            KeySchemaElement {
-                attribute_name: "rk2".to_string(),
-                key_type: "RANGE".to_string(),
-            },
-        ],
-        projection: Projection {
-            non_key_attributes: None,
-            projection_type: Some("ALL".to_string()),
-        },
-    }]);
-    create.table_name = table_name.to_string();
-    create.billing_mode = Some("PAY_PER_REQUEST".to_string());
-    client.create_table(create).await?;
-    Ok(())
+trait ErrExt {
```

Comment:
Is this here by mistake? It looks like you moved it to `lib.rs` and also down here.

Reply:
`ResultExt` is used in both `Backend`s but `ErrExt` is only used in the rusoto backend. It should also be noted that moving this is what causes the weird diff, since git is trying to interpret what happened and is very incorrect.